core/domain/platform_parameter_domain.py

```diff
@@ -19,7 +19,33 @@
 from __future__ import absolute_import  # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
+import re
+
+from constants import constants
 from core.domain import change_domain
+from core.platform import models
+import feconf
+import python_utils
+import utils
+
+
+(config_models,) = models.Registry.import_models(
+    [models.NAMES.config])
+memcache_services = models.Registry.import_memcache_services()
+
+SERVER_MODES = utils.create_enum('dev', 'test', 'prod') # pylint: disable=invalid-name
```

Comment:
If not, pylint yields error for this type of variable names ALL_UPPERCASE_WITH_UNDERSCORE. Here are similar cases
https://github.com/oppia/oppia/blob/a959629496ccf5213ca182e5bec8944218d6641f/core/storage/base_model/gae_models.py#L40-L52

Reply:
If not, pylint yields error for this type of variable names ALL_UPPERCASE_WITH_UNDERSCORE. Here are similar cases
https://github.com/oppia/oppia/blob/a959629496ccf5213ca182e5bec8944218d6641f/core/storage/base_model/gae_models.py#L40-L52

Reply:
Ah, hmm it seems like we need to somehow configure pylint to not complain about enums since they are effectively constants. @DubeySandeep is this something that's possible to configure in pylint?

Reply:
Discussed with Chu directly--deciding not to block the PR on this restriction since there's existing precedent. @MegrezZhu please do follow up with @DubeySandeep regarding how we can fix this codebase-wide, though.